src/Icicle/Sea/IO/Psv/Input.hs

```diff
@@ -224,20 +231,31 @@ seaOfReadAnyFactPsv opts config states = do
 seaOfReadNamedFactDense :: Base.InputOpts -> SeaProgramState -> Doc
 seaOfReadNamedFactDense opts state
  = let attrib = getAttribute (stateAttribute state)
-       err    = vsep
-                [ "        return ierror_loc_format"
-                , "           ( time_ptr + time_size"
-                , "           , time_ptr"
-                , "           , \"%s: time is out of order: %.*s must be later than %.*s\""
-                , "           , \"" <> pretty attrib <> "\""
-                , "           , curr_time_size"
-                , "           , curr_time_ptr"
-                , "           , last_time_size"
-                , "           , last_time_ptr );"
-                ]
+       errs   = Base.SeaInputError
+                ( vsep
+                [ "return ierror_loc_format"
+                , "   ( time_ptr + time_size"
+                , "   , time_ptr"
+                , "   , \"%s: time is out of order: %.*s must be later than %.*s\""
+                , "   , \"" <> pretty attrib <> "\""
+                , "   , curr_time_size"
+                , "   , curr_time_ptr"
+                , "   , last_time_size"
+                , "   , last_time_ptr );"
+                ])
+                ( vsep
+                [ "return ierror_loc_tag_format"
+                , "   ( IERROR_LIMIT_EXCEEDED"
+                , "   , entity_ptr + entity_size"
+                , "   , entity_ptr"
+                , "   , \"%.*s - %.*s facts were dropped starting from here\\n\""
```

Comment:
This doesn't really say why they were dropped, not sure what to say instead, maybe something like:
`$entity contained an unreasonable number of $attribute facts, they have been dropped`